packages/nextjs/components/streamer/Guru.tsx

```diff
@@ -0,0 +1,146 @@
+import { type FC, useEffect, useState } from "react";
+import { Address } from "../scaffold-eth";
+import { CashOutVoucherButton } from "./CashOutVoucherButton";
+import { Address as AddressType, formatEther, parseEther, verifyMessage } from "viem";
```

Comment:
Did you not import `encodePacked`, `keccak256` and `arrayified` in purpose to not give too much clues to speedrunners to implement the Checkpoint 3, or just forgot it?

Reply:
Thanks! Updated [67ed995](https://github.com/scaffold-eth/se-2-challenges/pull/178/commits/67ed9953c3c30da67ffb6dbb369da426cc30b5b3)